src/Akeneo/Platform/Bundle/UIBundle/tests/front/unit/components/JobExecutionProgress.unit.tsx

```diff
@@ -33,8 +33,7 @@ abstract class BaseViewMock {
   }
 }
 
-jest.mock('@akeneo-pim-community/legacy-bridge/src/bridge/react', () => ({ReactView: BaseViewMock}));
-jest.mock('oro/translator', () => (key: string, _params: any, count: number): string => {
+const translator = jest.fn().mockImplementation((key: string, _params: any, count: number): string => {
```

Comment:
```suggestion
const translator = jest.fn().mockImplementation((key: string, params: any, count: number): string => {
```

underscore prefixed variable names tells typescript to ignore unused variables. Si here as it's used, you should remove the leading underscore ;)

Reply:
I didn't know i leaned something :smile: